projects/plugins/backup/readme.txt

```diff
@@ -1,7 +1,7 @@
 === Jetpack VaultPress Backup ===
 Contributors: automattic, bjorsch, fgiannar, initsogar, jeherve, jwebbdev, kraftbj, macbre, samiff, sermitr, williamvianas
 Tags: jetpack, backup, restore
-Requires at least: 6.2
+Requires at least: 6.3
 Requires PHP: 5.6
```

Comment:
Should we update all these "Requires PHP" lines right away too?

Reply:
I was hoping #34126 would get merged before this one, that should help.